app/Models/User.php

```diff
@@ -23,6 +23,7 @@ class User extends Authenticatable
     const ROLE_SUPER = 'super admin';
     const ROLE_ADMIN = 'admin';
     const ROLE_CONTENT = 'Content Moderator';
+    const ROLE_SUPERID = 1;
```

Comment:
Change it to ROLE_ID_SUPER_ADMIN
Define names that are easy to understand. Also for ROLE_CONTENT, It should be ROLE_CONTENT_ADMIN and for super admin ROLE_SUPER_ADMIN

Reply:
Fixed